lib/plugins/aws/provider/awsProvider.js

```diff
@@ -364,9 +429,12 @@ class AwsProvider {
             },
             additionalProperties: false,
           },
-          awsLogGroupName: {
+          awsS3BucketName: {
             type: 'string',
-            pattern: '^[/#A-Za-z0-9-_.]+$',
+            pattern:
+              '(?!^(\\d{1,3}\\.){3}\\d{1,3}$)(^(([a-z0-9]|[a-z0-9][a-z0-9-]*[a-z0-9])\\.)*([a-z0-9]|[a-z0-9][a-z0-9-]*[a-z0-9])$)',
```

Comment:
What's the source for this regex?

Reply:
https://stackoverflow.com/questions/50480924/regex-for-s3-bucket-name

Reply:
Thanks! Can you add link to this page in comment above this `pattern`. I'm pretty sure others may also wonder about the source of that, and that will help to connect the dots